templates/api.html

```diff
@@ -10,7 +10,8 @@ <h2>Available fields</h2>
 		<div style="float: right; width: 60%; padding-left: 10px;">
 			<h2>Examples</h2>
 			{% for example in $resource.examples %}
-				<h3 id="{{ example.id }}">{{ example.summary}}</h3>
+				<h3 id="{{ example.id }}">{{ example.summary }}</h3>
+				<p>{{ example.description }}</p>
```

Comment:
I didn't test, just a thought: In case many of the descriptions were to be initially left empty, we wouldn't want this `<p>` tag to introduce a significant vertical blank space above the example box. Just wondering whether the `<div>` tag might introduce less vertical blank space. Something to test - it may not make any difference, and the issue may be imaginary, in my head only. :-)

Reply:
@pdcastro Yes, initially my thoughts were to document all the necessary descriptions myself but I think its too much work for myself to take on + a heavier PR will become harder to review. Instead, I will just drop the mechanism in place and get the PR ready with some I think we really need. 

Regarding your theory, I agree a <p> tag might leave some empty space if not dead HTML code in our website which ain't desirable. I will see what I can do about that.

Reply:
Good news, resolved it with a if statement that checks if there is a description in the first place if no `<p>` tag created.